spec/models/pair_request_spec.rb

```diff
@@ -2,17 +2,36 @@
 #
 # Table name: pair_requests
 #
-#  id          :bigint           not null, primary key
-#  duration    :float
-#  when        :datetime
-#  created_at  :datetime         not null
-#  updated_at  :datetime         not null
-#  acceptor_id :integer
-#  author_id   :integer
+#  id         :bigint           not null, primary key
+#  duration   :float            not null
+#  status     :integer          default(0), not null
+#  when       :datetime         not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  author_id  :integer          not null
+#  invitee_id :integer          not null
 #
 
 require 'rails_helper'
 
 RSpec.describe PairRequest do
-  
+  it "is valid with a duration longer than 15 minutes" do
```

Comment:
tonight i'll hop on and change this a bit. I have formatted a little differently when I do these (and you can let me know if we want to go this route). The tests themselves are good though.